src/sorcha/ephemeris/simulation_driver.py

```diff
@@ -80,6 +80,8 @@ def create_ephemeris(orbits_df, pointings_df, args, sconfigs):
         nside : integer
             The nside value used for the HEALPIx calculations.  Must be a
             power of 2 (1, 2, 4, ...)  nside=64 is current default.
+        n_sub_intervals: int
```

Comment:
can't you just set that 101 and be done with it as we're not changing it or having it read in my the config file? I guess the current implementation allows it to be set in the config if needed?